server/prisma/seedData.ts

```diff
@@ -0,0 +1,52 @@
+export const sampleNewbieTasks = [
```

Comment:
Please change this file name from `camelCase` to `kebab-case`

Reply:
Good catch. Fixed here: d55a2ae9a2ece6189a0e856811acfec6dadde6f1